src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -97,25 +97,26 @@ public function prepare_items() {
 
 		$args['posts_per_page'] = $per_page;
 
-		if ( 'scheduled' === $this->tab->get_slug() && ! empty( $_REQUEST['s'] ) ) {
+		if ( 'scheduled' === $this->tab->get_slug() && ! empty( tribe_get_request_var( 's' ) ) ) {
 			// nonce check if search form submitted.
 			$nonce = isset( $_POST['s'] ) && isset( $_POST['aggregator']['nonce'] ) ? sanitize_text_field( $_POST['aggregator']['nonce'] ) :  '';
-			if ( wp_verify_nonce( $nonce,  'aggregator_' . $this->tab->get_slug() . '_request'  ) ) {
+			if ( isset( $_GET['s'] ) || wp_verify_nonce( $nonce, 'aggregator_' . $this->tab->get_slug() . '_request' ) ) {
+				$search_term = filter_var( tribe_get_request_var( 's' ), FILTER_VALIDATE_URL ) ? esc_url_raw( tribe_get_request_var( 's' ) ) : sanitize_text_field( tribe_get_request_var( 's' ) );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 8 spaces but found 1 space

<sub>hash: 2bfe2e72aacbca3cb5f75ed54346e5bf</sub>

Reply:
Not sure if this still applies now that code is split to multiple lines